src/resources/language/_language-zh.yml

```diff
@@ -0,0 +1,62 @@
+toc-title-document: "目录"
+toc-title-website: "在该页面"
+
+section-title-footnotes: "脚注"
+
+callout-tip-caption: "提示"
+callout-note-caption: "注释"
+callout-warning-caption: "警告"
+callout-important-caption: "重要"
+callout-danger-caption: "危险"
+
+code-summary: "代码"
```

Comment:
^ same for this one -- my current translation just says "code" instead of "code summary", and then the one below also just says "code"

Reply:
The `code-tools-menu-caption` appears here:

<img width="1163" alt="Screen Shot 2021-11-08 at 9 15 53 PM" src="https://user-images.githubusercontent.com/104391/140849416-815d5fd1-28f3-41af-896a-bbcc19f9fb4d.png">

Reply:
`code-summary` is the summary text for the `<details>` tag used for folded code blocks:

<img width="693" alt="Screen Shot 2021-11-08 at 9 16 37 PM" src="https://user-images.githubusercontent.com/104391/140849523-bf3ad803-41c0-47a1-8b26-c49edae13b80.png">

Reply:
Cool. Then the current translations for both of those are fine.